src/debugger/gui/CartCDFInfoWidget.cxx

```diff
@@ -23,16 +23,28 @@ CartridgeCDFInfoWidget::CartridgeCDFInfoWidget(
     int x, int y, int w, int h, CartridgeCDF& cart)
   : CartDebugWidget(boss, lfont, nfont, x, y, w, h)
 {
-  uInt16 size = 8 * 4096;
-
+  uInt32 size;
   ostringstream info;
-  info << describeCDFVersion(cart.myCDFSubtype) << " cartridge\n"
-  << "32K ROM, seven 4K banks are accessible to 2600\n"
-  << "8K CDF RAM\n"
-  << "CDF registers accessible @ $FFF0 - $FFF3\n"
-  << "Banks accessible at hotspots $FFF5 to $FFFB\n"
-  << "Startup bank = " << cart.startBank() << "\n";
 
+  if (cart.myCDFSubtype == CartridgeCDF::CDFSubtype::CDFJplus) {
+    size = 512 * 1024;
+    info << describeCDFVersion(cart.myCDFSubtype) << " cartridge\n"
+      << "512K ROM (seven 4K banks are accessible to 2600)\n"
+      << "32K RAM\n"
+      << "Functions accessible @ $FFF0 - $FFF3\n"
+      << "Banks accessible @ $FFF4 to $FFFB\n"
+      << "Startup bank = " << cart.startBank() << "\n";
+
+  } else {
+    size = 8 * 4096;
+    info << describeCDFVersion(cart.myCDFSubtype) << " cartridge\n"
+      << "32K ROM (seven 4K banks are accessible to 2600)\n"
+      << "8K RAM\n"
+      << "Functions accessible @ $FFF0 - $FFF3\n"
+      << "Banks accessible @ $FFF4 to $FFFB\n"
```

Comment:
"...$FFF**5** to ... " Right?

Reply:
Actually $FFF4 is also mapped to bank 6 in CDFJ.

Reply:
Sorry, I meant to comment the CDFJ+ hotspots. They seem wrong.

Reply:
For CDFJ the hotspots are $FFF5 to $FFFB (with bank 6 as startup)
For CDFJ+ they are $FFF4 to $FFFA (with bank 0 as startup)
I would have preferred to keep them the same, but something is hitting $FFFB on startup, meaning it has to map to the startup bank.

Reply:
Yup, but in the description the cdfj+ hotspots are copied from cdfj.